api/router/router.go

```diff
@@ -30,6 +32,14 @@ import (
 	"github.com/runfinch/finch-daemon/version"
 )
 
+var errRego = errors.New("error in rego policy file")
```

Comment:
nit: not blocking, but should the router really know anything about rego? Seems like the router should take in arbitrary middleware and the daemon should do the configuration.